client/extensions/woocommerce/app/settings/shipping/shipping-zone-list.js

```diff
@@ -10,74 +11,68 @@ import { localize } from 'i18n-calypso';
-				locationDescription: '50 states',
-				methods: [ {
-					name: 'USPS',
-					description: 'All domestic services',
-				}, {
-					name: 'Flat Rate',
-					description: 'Minimum spend: $100',
-				} ],
-				icon: 'location'
-			}, {
-				locationName: 'Rest of the world',
-				locationDescription: '240 countries',
-				methods: [ {
-					name: 'USPS',
-					description: 'All international services',
-				} ],
-				icon: 'globe'
-			} ],
+const ShippingZoneList = ( { site, siteId, loaded, shippingZones, translate } ) => {
+	const renderContent = () => {
+		const renderShippingZone = ( zone, index ) => {
+			return ( <ShippingZoneEntry key={ index } siteId={ siteId } loaded={ loaded } { ...zone } /> );
 		};
-	}
-
-	renderShippingZone( zone, index ) {
-		return ( <ShippingZone key={ index } { ...zone } /> );
-	}
 
-	render() {
-		const { translate } = this.props;
-
-		const onAddZoneOpen = () => {
-			this.setState( { showDialog: true } );
-		};
-
-		const onAddZoneClose = () => {
-			this.setState( { showDialog: false } );
-		};
+		const zonesToRender = loaded ? shippingZones : [ {}, {}, {} ];
 
 		return (
 			<div>
-				<ExtendedHeader
-					label={ translate( 'Shipping Zones' ) }
-					description={ translate( 'The regions you ship to and the methods you will provide.' ) }>
-					<Button onClick={ onAddZoneOpen }>{ translate( 'Add zone' ) }</Button>
-				</ExtendedHeader>
-				<Card className="shipping__zones">
-					<div className="shipping__zones-row shipping__zones-header">
-						<div className="shipping__zones-row-icon"></div>
-						<div className="shipping__zones-row-location">{ translate( 'Location' ) }</div>
-						<div className="shipping__zones-row-methods">{ translate( 'Shipping methods' ) }</div>
-						<div className="shipping__zones-row-actions" />
-					</div>
-					{ this.state.shippingZones.map( this.renderShippingZone ) }
-				</Card>
-				<ShippingZoneDialog isVisible={ this.state.showDialog } onClose={ onAddZoneClose } />
+				<div className="shipping__zones-row shipping__zones-header">
+					<div className="shipping__zones-row-icon"></div>
+					<div className="shipping__zones-row-location">{ translate( 'Location' ) }</div>
+					<div className="shipping__zones-row-methods">{ translate( 'Shipping methods' ) }</div>
+					<div className="shipping__zones-row-actions" />
+				</div>
+				{ zonesToRender.map( renderShippingZone ) }
 			</div>
 		);
-	}
-}
+	};
+
+	const addNewHref = loaded
+		? getLink( '/store/settings/shipping/:site/zone/new', site )
+		: '#';
+
+	const onAddNewClick = ( event ) => {
+		if ( ! loaded ) {
+			event.preventDefault();
+		}
+	};
+
+	return (
+		<div>
+			<QueryShippingZones siteId={ siteId } />
+			<ExtendedHeader
+				label={ translate( 'Shipping Zones' ) }
+				description={ translate( 'The regions you ship to and the methods you will provide.' ) }>
+				<Button
+					href={ addNewHref }
+					onClick={ onAddNewClick }
+					disabled={ ! loaded }>{
+					translate( 'Add zone' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **9** times:
`translate( 'Add DNS Zone' )` *ES Score: 9*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/14275-shipping-methods-ui)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).